extension.Rmd

```diff
@@ -0,0 +1,241 @@
+         federalism = lfed1, # Decentralization
+         z_asylumseeker = zasylumseekers ,
+         z_unemp = zsur, # Unemployment rate
+         z_replacement = zreplacementrate, # Unemp benefits
+         max_er = rmax) # Maximum toughness in extreme right discourse
+```
+
+
+# Preparing data for analysis
+
+```{r}
+# Create centered variables for Saliance and Variance based on grand mean centering according to author
+
+data <- data  %>% 
+  mutate(salience_mean_c = (salienzmean - 3.84568)) %>% 
+  mutate(var_c = rvar - 21.75423)
+
+# Make male variable numeric
+data <- data %>%
+  mutate(male = as.numeric(male))
+
+# Create a dummy variable for country (transform from character to factor variable)
+data <- data %>%
+  mutate(country = factor(sortcountry, labels = c("AT", "BE","DE-E","DE-W","DK","ES","FI","FR","GR","IT","LU","NL","NO","PT","SE")))
+```
+
+
+# Model 1: Adorno Thesis (Individual Characteristics and Value Orientations)
+The first model is based on Theodor Adorno's 1950 theory that support for extreme right parties (in his case, fascist parties) originates from supporters' personality traits and value orientations, which make individuals more receptive to the appeals of extreme right parties.  We operationalize "personality traits and value orientations" using the individual level characteristics from our data set: gender, age, occupation, marital status, and self-placement on a left-right scale. These variables are rather poor proxies for "personality traits and value orientations", but provide the closest approximation from our dataset.
+
+```{r}
+model_1 <- glm(ext_vote ~ male + age_1 + age_2 + age_4 + edu_1 + edu_2 + farmer_own + worker + retired + unemployed + partner + left_right_scale + country - 1, data = data)
+summary(model_1)
+actual <- model_1$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_1$fitted.values # Create a vector with log-odd values
+# Create link function to transform log-odds into predicted probabilities
+link <- function(log.odd) {
+  pred.prob <- 1/(1+exp(-log.odd))
+  pred.prob # returns a numeric vector of predicted probabilities
+} 
+pred_prob <- link(log.odds) # Transform log-odds into predicted probabilities
+pred_prob <- as.vector(pred_prob) 
+range(pred_prob) # variation in the predicted probabilities: very little variation.
+# Graph variation in Predicted probabilities
+pred_df <- data.frame(pred_prob)
+ggplot(pred_df, aes(x = 1:length(pred_prob), y = pred_prob)) + 
+  geom_line() +
+  xlab("Prediction Number") +
+  ylab("Predicted Probability") +
+  ggtitle("Model 1: Variation in Predicted Probabilities")
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred_prob, actual = actual, type = "rect", heading = "Model1: The Adorno Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 1](figures/SP.Model1.png)
```

Comment:
did you notice the clear structure in the plot?  Are the cases sorted by country?